Mathlib/RingTheory/IntegralClosure/Algebra/Basic.lean

```diff
@@ -68,6 +68,10 @@ variable (R B) in
 instance Algebra.IsIntegral.of_finite [Module.Finite R B] : Algebra.IsIntegral R B :=
   ⟨.of_finite R⟩
 
+lemma Algebra.IsIntegral.of_surjective_algebraMap (H : Function.Surjective (algebraMap R B)) :
+    Algebra.IsIntegral R B :=
+  .of_surjective (IsScalarTower.toAlgHom R R B) H
```

Comment:
Is this different from [Algebra.isIntegral_of_surjective](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/IntegralClosure/IsIntegralClosure/Basic.html#Algebra.isIntegral_of_surjective)?
By the way, there is [Algebra.ofId](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Algebra/Hom.html#Algebra.ofId) you can use.